cabot_alert_pagerduty/models.py

```diff
@@ -48,6 +50,7 @@ def send_alert(self, service, users, duty_officers):
                                              service.overall_status)
 
         failed_checks = [check.name for check in service.all_failing_checks()]
+        runbook_contents = {check.name : _get_runbook_content_for_check(check.name) for check in failed_checks}
```

Comment:
failed_checks already filters for the check.name so this line will break

Reply:
Thank you for the catch. Updated